tests/integration/targets/meraki_ms_l3_interface/tasks/main.yml

```diff
@@ -3,59 +3,64 @@
 
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 ---
-- block:
+- name: Block of tests
+  block:
   - name: Create network with type switch
-    meraki_network:
+    cisco.meraki.meraki_network:
       auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkSwitch
       type: switch
       timezone: America/Chicago
     delegate_to: localhost
     register: create_net_switch
 
   - name: Claim a device into a network
-    meraki_device:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
+    cisco.meraki.meraki_device:
+      auth_key: '{{ auth_key }}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkSwitch
       serial: '{{ serial_switch_l3 }}'
       state: present
     delegate_to: localhost
 
   - name: Query all l3 interfaces
-    meraki_ms_l3_interface:
+    cisco.meraki.meraki_ms_l3_interface:
       auth_key: '{{ auth_key }}'
       state: query
       serial: '{{ serial_switch_l3 }}'
     delegate_to: localhost
     register: query_all
 
-  - set_fact:
+  - name: Set fact interface_id_1
```

Comment:
This is failing because the switch isn't showing any interfaces configured.

```
TASK [meraki_ms_l3_interface : Query all l3 interfaces] ************************
ok: [localhost]

TASK [meraki_ms_l3_interface : debug] ******************************************
ok: [localhost] => {
    "query_all": {
        "changed": false,
        "data": [],
        "failed": false,
        "response": "OK (unknown bytes)",
        "status": 200,
        "warnings": [
            "Platform darwin on host localhost is using the discovered Python interpreter at /usr/local/bin/python3.11, but future installation of another Python interpreter could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information."
        ]
    }
}
```

We'll probably need to configure something to get this to work. It worked before and I'm not sure why though.

Reply:
@kbreit I am going to debug it when I got the DevNet SandBox instance.

Reply:
Hello! @kbreit, I got the new devs env (Dev SandBox env and Meraki Demo) in that envs I do not have a L3 Switch. I will write to dev support maybe I will get the new switch in my test envs for debug it.

Reply:
I got the access to the test devices. But all of that devices it is a L2 switches. I will write again to the support for the L3 switch to try resolve the issue.

Reply:
Good news. I was able to resolve this without adding anything. Move the "Create l3 interface" tasks and assertions, including check mode, above the query all task.

Reply:
Hello!
Can you give what I should move up due to I still got the error?

Reply:
Move these lines up so they fall just after the task which claims the device into the network.\

- https://github.com/CiscoDevNet/ansible-meraki/blob/ab2a4b3b117cd5c33934e67e0732bd6124c12acc/tests/integration/targets/meraki_ms_l3_interface/tasks/main.yml#L57
- https://github.com/CiscoDevNet/ansible-meraki/blob/ab2a4b3b117cd5c33934e67e0732bd6124c12acc/tests/integration/targets/meraki_ms_l3_interface/tasks/main.yml#L76
- https://github.com/CiscoDevNet/ansible-meraki/blob/ab2a4b3b117cd5c33934e67e0732bd6124c12acc/tests/integration/targets/meraki_ms_l3_interface/tasks/main.yml#L81
- https://github.com/CiscoDevNet/ansible-meraki/blob/ab2a4b3b117cd5c33934e67e0732bd6124c12acc/tests/integration/targets/meraki_ms_l3_interface/tasks/main.yml#L99
- https://github.com/CiscoDevNet/ansible-meraki/blob/ab2a4b3b117cd5c33934e67e0732bd6124c12acc/tests/integration/targets/meraki_ms_l3_interface/tasks/main.yml#L102